### ISSUE TITLE
Does reversing suffixes for kb_base of 1024 really make sense?

### ISSUE THREAD

**Issue by horshack-dpreview** [axboe/fio#1396]
**Description of the new feature**

When kb_base==1024 (default), all EC 80000-13 suffixes like KiB and MiB are interpreted as their ^10 equivalents, which matches the [fio documentation](https://fio.readthedocs.io/en/latest/fio_man.html#cmdoption-arg-kb-base). However does this make sense? It's logical that the ^10 suffixes like KB and MB are re-interpreted as KiB and MiB, since the stated purpose of kb_base==1024 is for backward compatibility with old scripts that weren't aware of EC 80000-13 suffixes. But are there also old scripts that used KiB and MiB expecting them to translate to ^10 instead of ^2 values? If not, does it make more sense for EC 80000-13 suffixes to always be interpreted as ^2 irrespective of kb_base, and have kb_base==1024 only change the non-EC 80000-13 suffixes suffixes like KB and MB?

To recap:

Current behavior:
kb_base==1024: KiB translates to 1000, KB translates to 1024
kb_base==1000: KiB translates to 1024, KB translates to 1000

Proposed new behavior:
kb_base==1024: KiB translates to 1024, KB translates to 1024
kb_base==1000: KiB translates to 1024, KB translates to 1000
